cmd/byohctl/service/templates/agent-systemd.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Script to set up systemd service for BYOH agent
+
+BYOH_DIR="$1"
+AGENT_BINARY="$2"
+NAMESPACE="$3"
+LOG_DIR="$BYOH_DIR/logs"
+AGENT_LOG="$LOG_DIR/agent.log"
+
+# Create systemd service file
+cat > /etc/systemd/system/byoh-agent.service << EOF
```

Comment:
Why is this script generating the service file when you have a similar template file above doing the same thing?

Reply:
I forgot to remove these scripts. Removed them in this commit